2025/using-components-2/demo/2-react/vite.config.js

```diff
@@ -0,0 +1,9 @@
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
```

Comment:
There is a faster and newer alternative to this plug-in: https://github.com/maxpatiiuk/esri-dev-summit-presentations/blob/224034c97f9a02abac37354d493ac008ee2407b8/2025/build-tooling/demo/2-react/vite.config.ts#L3